time/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+//! A safe date and time library for Rust.
+//!
+//! [The book] is the main documentation for `time`.
```

Comment:
This definitely isn't the case. Right now the only thing it's the primary reference for is format descriptions. Most of the book is still empty, as I've focused my efforts elsewhere.

Reply:
Would you prefer a different text here, or perhaps a badge? I do recall struggling with format descriptions before finding the book.

Reply:
In that specific location, I think it could go without until there's more content. However, adding a reference to the relevant pages of the book on the various format/parse methods would make sense.

Reply:
I'll remove this change for this PR, the format/parse methods are reasonably documented IMO.